requirements.txt

```diff
@@ -0,0 +1,4 @@
+cython>=3.0.0
+numpy>=1.10.0
+greenlet>=3.0.0
+cmake
```

Comment:
Maybe add a comment to the cmake line that cmake is needed for the Charm++ build.

Reply:
done